### PR TITLE
[Campinas-2024]  Adding ediction dod São Paulo 2024

### PR DESCRIPTION
**Pull request by GiovanaNp1**
This is the first commit for starting the jobs in DevOpsDays Campinas

### CODE REVIEW

File: data/events/2024/campinas/main.yml [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  - name: "Flavio Pimenta"
+    role: "Software Engineer"
+    twitter: "pimentaflavio"
+    bio: "Arquiteto de Soluções focado em traduzir necessidades de negócio em soluções técnicas robustas e com alta qualidade de ponta a ponta.."
+    image: "pimenta.jpeg"
+    github: "https://github.com/flap"
+    linkedin: "https://www.linkedin.com/in/flaap/?originalSubdomain=br"
+    website: "https://campsite.bio/pimenta/"
+    
+organizer_email: "devopsdays.campinas@gmail.com" # Put your organizer email address here
```
Comment (phrawzty) [inline data/events/2024/campinas/main.yml:95]: This _must_ be your `@devopsdays.org` address.

```suggestion
organizer_email: "campinas@devopsdays.org" # Put your organizer email address here
```

File: data/events/2024/campinas/main.yml [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    url: /events/2024-campinas/conduta 
+  # - name: requisitos para as palestras
+  #   url: /events/2024-campinas/propose 
+# - name: example
+#   icon: "map-o" # This is a font-awesome icon that will display on small screens. Choose at http://fontawesome.io/icons/
+#   url: http://mycfp.com # The url setting is optional, and only if you want the navigation to link off-site
+
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
```
Comment (phrawzty) [inline data/events/2024/campinas/main.yml:55]: The list here doesn't appear to line up with the current `campinas@devopsdays.org` distribution list. Please send an email to info@devopdays.org with the full list of names and email addresses of your local team (it must match this list here in `main.yml`).

**Review by phrawzty — CHANGES_REQUESTED** [review]

File: data/events/2024/campinas/main.yml (hunk 2 above)
Comment (GiovanaNp1) [reply data/events/2024/campinas/main.yml:55]: Hello @phrawzty i try send email for [info@devopdays.org](mailto:info@devopdays.org) but i I received this response

Comment (GiovanaNp1) [reply data/events/2024/campinas/main.yml:55]: @phrawzty  could help me with this topic?

Comment (phrawzty) [reply data/events/2024/campinas/main.yml:55]: @GiovanaNp1 You spelled `devopsdays` incorrectly—you're missing the `s` in `ops`. 🫢

Comment (GiovanaNp1) [reply data/events/2024/campinas/main.yml:55]: Sorry my mistake, i'm send now the update

File: data/events/2024/campinas/main.yml [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+#description: "Em breve!" # Edit this to suit your preferences
+ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+
+# All dates are in unquoted 2024-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2024-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate: 2024-09-21T00:00:00-03:00 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate: 2024-09-21T23:59:59-03:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
```
Comment (gomex) [inline data/events/2024/campinas/main.yml:14]: A data só pode ser colocada no PR quando tiver o local confirmado.

**Review by gomex — CHANGES_REQUESTED** [review]
Pode confirmar se ja tem local? A data só pode ser colocada com local confirmado!

File: data/events/2024/campinas/main.yml [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    url: /events/2024-campinas/conduta 
+  # - name: requisitos para as palestras
+  #   url: /events/2024-campinas/propose 
+# - name: example
+#   icon: "map-o" # This is a font-awesome icon that will display on small screens. Choose at http://fontawesome.io/icons/
+#   url: http://mycfp.com # The url setting is optional, and only if you want the navigation to link off-site
+
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
```
Comment (phrawzty) [inline data/events/2024/campinas/main.yml:55]: n.b. Still waiting for clarity on team membership composition.

**Comment by GiovanaNp1** [comment]
I'm canceling this PR to get the new information I need

**Pull request closed without merging**